gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/oauth/OAuthPage.kt

```diff
@@ -172,7 +172,9 @@ internal fun OauthPage(
                         uiState.profile?.let {
                             ProfileCard(
                                 profile = it,
+                                email = email,
                                 modifier = Modifier.padding(top = 16.dp),
+                                avatarCacheBuster = uiState.avatarCacheBuster,
```

Comment:
We probably don't have to use the cache buster on this page, but:
- it makes testing easier, and
- it makes sure the the QE always shows the latest and correct avatar